docs/lib.md

```diff
@@ -48,6 +48,21 @@ utils.getDefaultAccounts();
 ```
 Get the pre-funded default accounts as `MemoryAccount`, so they can be used natively using the `aeSdk`.
 
+```javascript
+utils.getUsedFee(() => contract.methods.X(...));
+```
+Calls X and returns the total fee paid for the transaction
+
+```javascript
+utils.failsWith(() => contract.methods.X(...), 'errorMsg');
```

Comment:
Isn't it `await expect(contract.methods.X(...)).to.be.rejectedWith(ErrorClass, 'errorMsg')`?
https://www.chaijs.com/plugins/chai-as-promised/